File: help/pt_pt/exercicios_da_ficha_8/Ficha08_005/Ficha08_005.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha08_005 {

    //Main Function
    static void Main() {
        double altura = 0.0 ;
        do{
            Console.Write("Altura : " );
            altura = double.Parse(Console.ReadLine());
        }while(altura < 0.2 || altura > 3.0 );
        Console.Write("Altura = " + altura );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Thu, 2022 Sep 29 12:17

File: help/pt_pt/exercicios_da_ficha_8/Ficha08_005/Ficha08_005.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha08_005 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        double altura = 0.0 ;
        do{
            System.out.print("Altura : " );
            altura = Double.valueOf(keyb.nextLine());
        }while(altura < 0.2 || altura > 3.0 );
        System.out.print("Altura = " + altura );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Thu, 2022 Sep 29 12:17
